src/jit/compiler.cpp

```diff
@@ -49,6 +49,13 @@ AssemblyNamesList2* Compiler::s_pAltJitExcludeAssembliesList            = nullpt
 #endif // ALT_JIT
 
 /*****************************************************************************/
+#ifdef _TARGET_AMD64_
+#include <intrin.h>
+inline bool _our_GetThreadCycles(unsigned __int64 *cycleOut){ *cycleOut = __rdtsc(); return true; }
```

Comment:
Oh, thanks @janvorli - you saved me time! x-plat is not the highest pri right off the bat (and, as all of my PR's, everything is off by default), but I'd like to make it work as widely as possible. I'll definitely try to steal / reuse the stuff you mention.